### ISSUE TITLE
hsdev returns file not found/module not inspected error

### ISSUE THREAD

**Issue by hschroedl** [mvoidex/hsdev#35]
I am trying to get hsdev working with SublimeHaskell. However, hsdev returns only error codes, even when working from the command line. 

I tried a couple of things so far. See commands and error messages below:

`hsdev check -f /mnt/E69AFDCE9AFD9B6F/Downloads/double.hs`

`{"notify[{"status":"working","progress":null,"name":"/mnt/E69AFDCE9AFD9B6F/Downloads/double.hs","task":"scanning","type":"module"}]}
{"notify":[{"status":"ok","progress":null,"name":"/mnt/E69AFDCE9AFD9B6F/Downloads/double.hs","task":"scanning","type":"module"}]}
{"error":"module not inspected","module":{"project":null,"file":"/mnt/E69AFDCE9AFD9B6F/Downloads/double.hs"}}`

`hsdev whois -f /mnt/E69AFDCE9AFD9B6F/Downloads/double.hs x`

`{"error":"other error","msg":"File '/mnt/E69AFDCE9AFD9B6F/Downloads/double.hs' not found"}`

This corresponds to the errors I am getting in Sublime:

`Sublime Haskell: hsdev whois returns error: other error, id: 16, msg: File '/mnt/E69AFDCE9AFD9B6F/Downloads/double.hs' not found`
`Sublime Haskell: hsdev lookup returns error: other error, id: 17, msg: File '/mnt/E69AFDCE9AFD9B6F/Downloads/double.hs' not found`
`Sublime Haskell: hsdev types returns error: module not inspected, id: 15, module: {'file': '/mnt/E69AFDCE9AFD9B6F/Downloads/double.hs', 'project': None}`

Currently I am using `ghc-8.0.1`, `hsdev-0.2.1.0` and `cabal 1.22.6.0`. I am pretty new to Haskell, so I apologize if I am somehow missing important information. I actually had SublimeHaskell fully working last week, but since then I upgraded ghc, and it appears that I broke something. 

I realise that this is probably a problem on my end, which you wont be able to reproduce, but any help in regards how to debug this issue would be greatly appreciated.

**Comment by hschroedl** [comment]
Update: hsdev works as expected *as long as functions have been exposed using a module. 

```
module Spronk(
        funci)
where

funci :: Integer -> Integer
funci x = x+1
```

If the first couple of lines are deleted the error described above occurs. Is this behavior intended? If so, an update to the readme of this project (or SublimeHaskell's) might be in order.

**Comment by mvoidex** [comment]
No, it's a bug, thanks!

**Comment by YeshaMaggi** [comment]
I believe I have run into the same problem. The issue at hand for me is that errors never show, on save or by `shift + ctrl + e`. I'm posting here since the module exposing temp-solution mentioned above works for me as well. (Thankfully!) I was running Debian 8 before and didn't have this issue with SublimeHaskell, but I just installed Lubuntu 16.1 and then this came about.

I haven't tried running hsdev from the command line, but from sublime I get the error:
`Sublime Haskell: hsdev module returns error: module not inspected ...`

I tried it with `ghc 7.10.3` and `ghc 8. 0.2` with the same results. Both times with `cabal 1.22.8.0` and `hsdev-0.2.1.0`.
When I tried it with `hsdev-0.2.0.0`, `Sublime Haskell: Checking and Linting:` displays indefinitely. (Well my patience ran out after 20 minutes.)

`enable_ghc_mod` and `enable_hsdevtools` don't seem to make a difference to this.
If I set `"enable_hsdev": false` and use ghc-mod, there is also no propagation of Haskell errors. I'm not familiar with what all these tools do for Haskell, but on my Debian set-up I was running the SublimeHaskell version from `https://github.com/SublimeHaskell/SublimeHaskell/tree/hsdev`, and did not have hsdev enabled, and things were working to my usage (with some work-arounds of course!)

I can post Sublime's console errors that occur with `"enable_hsdev": false` if that is helpful to fixing this bug. I'm personally satisfied with the module trick, and just wanted to add any information from my usage that might be helpful for this issue.
